src/styles/header/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  display: flex;
+  box-sizing: border-box;
```

Comment:
The `box-sizing: border-box;` rule is usually applied globally to all elements at the beginning of your CSS file. This helps to avoid unexpected behavior in layout calculations due to differences in box-sizing.